dali/test/python/test_dali_cpu_only.py

```diff
@@ -984,8 +1172,14 @@ def test_separated_exec_setup():
     "math.fpow",
     "math.acosh",
     "math.min",
-    "pytorch.TorchPythonFunction",
     "numba.fn.experimental.numba_function",
+    "dl_tensor_python_function",
+]
+
+excluded_methods = [
+    "noise.gaussian",
+    "noise.salt_and_pepper",
```

Comment:
Those two should be tested now. Can you rebase and check?

Reply:
Done